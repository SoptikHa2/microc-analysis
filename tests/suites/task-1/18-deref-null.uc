 main() {
   var x;
   x = null;
   *x = 42;
   return *x;
 }
