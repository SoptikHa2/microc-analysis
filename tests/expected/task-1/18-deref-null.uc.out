---
test: run suites/task-1/18-deref-null.uc
---
microc: Error during interpretation.
At "suites/task-1/18-deref-null.uc" (line 3, column 13): Null-pointer dereference

exit: 1
